testing/integration-testing-hooks.ts

```diff
@@ -192,9 +212,9 @@ export function startTestServer(shareMode: 'shared' | 'not-shared', args: Partia
     if (Object.keys(args).length > 0) {
       throw new Error('Cannot specify arguments for shared mongod');
     }
-    server = sharedSetup ?? (sharedSetup = new MongoRunnerSetup());
+    server = sharedSetup ?? (sharedSetup = new MongoRunnerSetup(id));
```

Comment:
Including an id in this case doesn’t make much sense, does it? The point of the `shared` server is that it’s shared for all tests running in the same process and we don’t need to start up/tear down server instances each time, but that also means that assigning an `id` makes somewhat limited sense.

Reply:
Ah that's true, yeah! Makes sense, i'll skip and use 'shared' it in that case.

Reply:
@addaleax too much? https://github.com/mongodb-js/mongosh/pull/1611/commits/0b7a34493cfb610d2b6ed4d600403cf6715f95cb

Reply:
Ah i have to update the comments